cookbooks/travis_build_environment/recipes/ci_user.rb

```diff
@@ -31,12 +35,12 @@
 end
 
 [
-  { name: node['travis_build_environment']['home'] },
-  { name: "#{node['travis_build_environment']['home']}/.ssh" },
-  { name: "#{node['travis_build_environment']['home']}/builds", perms: 0o755 },
-  { name: "#{node['travis_build_environment']['home']}/.m2" },
-  { name: "#{node['travis_build_environment']['home']}/gopath" },
-  { name: "#{node['travis_build_environment']['home']}/gopath/bin" }
+  { name: home },
+  { name: home.join('.m2') },
+  { name: home.join('.ssh') },
+  { name: home.join('builds'), perms: 0o755 },
+  { name: home.join('gopath') },
+  { name: home.join('gopath/bin') }
```

Comment:
I wonder if I should move the `$HOME/bin` directory creation here from the packer build scripts?